README.md

```diff
@@ -3,6 +3,8 @@ LLM Performance Benchmarking
 
 ## Performance
 
```

Comment:
Let's have the following sections:
* Single GPU, 4-bit
* Multiple NVIDIA GPU, fp16
* Multiple AMD GPU, 4-bit

Reply:
Are we supposed to include Multiple NVIDIA GPU, 4-bit?